src/tsickle.ts

```diff
@@ -359,13 +388,15 @@ class Annotator extends ClosureRewriter {
     let annotated = this.getOutput();
 
     let externsSource: string|null = null;
-    if (externs.output) {
+
+    if (externs.output.length > 0 || this.exportingDecoratorExterns.length > 0) {
       externsSource = `/**
  * @externs
  * @suppress {duplicate}
  */
 // NOTE: generated by tsickle, do not edit.
-` + externs.output;
+` + externs.output +
```

Comment:
Just to make sure I'm following, you have code that ends up like:

.js: class Foo { bar() {} }
.externs.js: var bar;

And bar doesn't get renamed?  I am skeptical but I reproduced it locally so I guess it's right.  I'm not sure why this works though!